src/models/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .User import User
 from .Review import Review
-from .Drink import Drink
\ No newline at end of file
+from .Drink import Drink
+from .Single import SingleReview
```

Comment:
Right idea but this line doesn't belong here. This should go in `resources/single/__init__.py` and another in `resources/__init__.py`.